### ISSUE TITLE
Ability to have settings live in a custom directory

### ISSUE THREAD

**Issue by JohannesMP** [dziemborowicz/hourglass#217]
Configuration settings for Hourglass seem to currently be in a somewhat unintuitive directory in `AppData`:

![image](https://user-images.githubusercontent.com/662874/135926510-629780f5-6f8d-42b7-9d27-fb590d475ccd.png)

I'd like to be able to set a directory to be used instead, so I can have better control over the settings without having to resort to something like Symlinking.

Also it would also be useful if the directory used for storing config data was in a more intuitive directory than `Hourglass.exe_Url_la3v0chg0cfk1mkapgxkzi2xaouqgqcm`

**Comment by dziemborowicz** [comment]
Yeah. That's the default directory that the settings API gives you in .NET Framework. I have no idea why... (Well, I do, but still, it's not great.)

The portable version of Hourglass stores the settings in a file that is a sibling to the .exe if that would work better for you.

**Comment by Micke3rd** [comment]
Hi dziemborowicz,
the currently widespread approach is to deploy only a portable version, ready for the argument "data-dir"
e.g. --data-dir=%appdata%\Whatever

In case of HourGlas I dont needed it, but what's more important for me, this would presumably make your addional setup build unnecessary.
I came in touch with your multiple builds, when I wanted support the project with a push to .Net 6. But I dont wanna dive into code with installers, because I dont like this way of deployment.
